MVP/jenkins-jobs.sh

```diff
@@ -26,4 +26,4 @@ fi
 
 set -ex
 upgrade_jjb
-jenkins-jobs --ignore-cache --conf config.ini $1 .
+~/.local/bin/jenkins-jobs --ignore-cache --conf config.ini $1 .
```

Comment:
It's good thinking to solve the issue when some user env doesn't have use binaries path in $PATH. However this script is designed to work on multiple platforms which may use different path for user binaries - thus using absolute path may break things.
How about updating $PATH instead if path to local user binaries is missing? E.g. `export $PATH:$HOME/.loca/bin`